libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -222,6 +233,11 @@ export class RushConfiguration {
   // Lazily loaded when the projectsByTag() getter is called.
   private _projectsByTag: ReadonlyMap<string, ReadonlySet<RushConfigurationProject>> | undefined;
 
+  // Cache subspace projects
+  private _subspaceProjectsCache: Map<string, RushConfigurationProject[]>;
```

Comment:
```suggestion
  private _rushProjectsBySubspaceNameCache: Map<string, RushConfigurationProject[]>;
```

Reply:
@iclanton our naming convention is `<name of map value>By<name of map key>`.  So if we want to give more adjectives about the value, it should go on the front, like `_cachedRushProjectsBySubspaceName`.

@D4N14L @william2958

Reply:
changed to _cachedRushProjectsBySubspaceName